packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -408,24 +412,25 @@ describe("<DateRangeInput>", () => {
             const { root } = wrap(<DateRangeInput {...{ startInputProps, endInputProps }} />);
             root.setState({ isOpen: true });
 
-            const startInput = getStartInput(root);
-            startInput.simulate("focus");
-            startInput.simulate("change", { target: { value: START_STR } });
-            startInput.simulate("keydown", { which: Keys.ENTER });
+            // Don't save the input elements into variables; they can become
+            // stale across React updates.
+
+            getStartInput(root).simulate("focus");
+            getStartInput(root).simulate("change", { target: { value: START_STR } });
+            getStartInput(root).simulate("keydown", { which: Keys.ENTER });
```

Comment:
you might be able to chain these simulates and only get a new reference on line 426?

Reply:
Chatted IRL. Will leave it b/c we're not sure exactly if the input node would get stale between events.